contrib/verifybinaries/verify.py

```diff
@@ -59,124 +194,457 @@ def download_with_wget(remote_file, local_file=None):
     return result.returncode == 0, result.stdout.decode().rstrip()
 
 
+def download_lines_with_urllib(url) -> (bool, [str]):
+    """Get (success, text lines of a file) over HTTP."""
+    try:
+        return True, [
+            line.strip().decode() for line in urllib.request.urlopen(url).readlines()]
+    except urllib.request.HTTPError:
+        log.warning(f"HTTP request to {url} failed (HTTPError)")
+    except Exception as e:
+        log.warning(f"HTTP request to {url} failed ({e})")
+    return (False, [])
+
+
+def verify_with_gpg(
+    signature_filename,
+    output_filename: t.Optional[str] = None
+) -> t.Tuple[int, str]:
+    # Enforce English output since we're relying on output that contains natural language.
+    # Use string-shell form (vs. args as list) because passing LANGUAGE via `env` doesn't
+    # seem to work.
+    args = (
+        'LANGUAGE=en gpg --yes --decrypt '
+        + (f' --output {output_filename} ' if output_filename else '')
+        + signature_filename)
```

Comment:
Previous version of this that used an arg list seemed better, because it avoided shell syntax that wouldn't work on windows and would have trouble with spaces and special characters.

I don't think there should be an issue passing the LANGUAGE value if you just add `, env=dict(os.environ, LANGUAGE="en")` to the subprocess call.